ci/deps/actions-pypy-38.yaml

```diff
@@ -0,0 +1,20 @@
+name: pandas-dev
+channels:
+  - conda-forge
+dependencies:
+  # TODO: Add the rest of the dependencies in here
+  # once the other plentiful failures/segfaults
+  # with base pandas has been dealt with
+  - python=3.8
+
+  # tools
+  - cython>=0.29.24
+  - pytest>=6.0
+  - pytest-cov
+  - pytest-xdist>=1.31
+  - hypothesis>=5.5.3
+
+  # required
+  - numpy<1.20  # GH#39541 compat for pyarrow<3
```

Comment:
id rather have both newer pyarrow and numpy, why are you doing it the other way around?

Reply:
Thanks for the catch. I accidentally forgot to change this when I copy-pasted from the database builds.